### ISSUE TITLE
VBA qualifier for the Mid statement result in a Type mismatch

### ISSUE THREAD

**Issue by FullValueRider** [twinbasic/twinbasic#1095]
**Describe the bug**
Having been well and truly bitten in the past I routinely prefix VBA native methods with the VBA prefix.  Today, for the first time ever, I had occasion to use the Mid statement (Note not the Mid function).  Prefixing the Mid statement with VBA Gives a Type mismatch error
**To Reproduce**
```VBA
Public Sub TestMidStatement()
    Dim myString As String = "HellowWorld"
    VBA.Mid(myString, 6, 1) = " "
    Debug.Print myString
End Sub

```

![ErrorOnMidStatement](https://user-images.githubusercontent.com/81257042/183647769-320b673c-f495-40ab-b5cd-87455f56967a.jpg)

If the VBA qualifier is omitted the code works as expected and prints "Hello World"

**Expected behavior**
The VBA qualifier should be respected for the Mid statement.

**Desktop (please complete the following information):**
 - OS: Windows 11
 - twinBASIC compiler version IDE Beta 90

**Comment by wqweto** [comment]
JFYI, `VBA.Mid(myString, 6, 1) = " "` fails in VB6 with a run-time error.

Using fully-qualified `VBA.Mid` instead of the intrinsic `Mid` statement leads to compiler producing code which is expecting an object in the Variant returned my `VBA.Mid` function with `Property Let` for the default property being implemented so it bombs with run-time error `424 Object required` as the Variant returned by `VBA.Mid` is (in 100% of the cases) a String.

In short: not a bug in TB and failing during compilation is even preferred although error message can be customized so to be clear that here `Mid` statement cannot be prefixed by `VBA` as this changes the semantics of the expression.

**Comment by Peter9Claudius** [comment]
If the problem once existed, it now appears to be fixed. For me the variants "vba.mid(...)", "mid(...)" and "mid$(...)" give correct result...
![grafik](https://user-images.githubusercontent.com/109410432/184477187-ea179ba9-ea80-43d3-8032-acd0b2c2d552.png)
![grafik](https://user-images.githubusercontent.com/109410432/184477219-6f833a86-9fa1-43b3-bf7c-e01c14f82930.png)
...genau so wie in VBA:
![grafik](https://user-images.githubusercontent.com/109410432/184477237-43a13d94-1c25-41d4-827f-937702e0f11e.png)
![grafik](https://user-images.githubusercontent.com/109410432/184477260-a978f666-9d82-42f8-bb3d-6655e27a0def.png)

**Comment by WaynePhillipsEA** [comment]
This now generates a compiler error in BETA 93, thanks!
